src/utils/colors-utils.ts

```diff
@@ -3,7 +3,11 @@ import { COLOR_STYLES, contentColors } from "./colors-vars-map";
 const ColorUtils = {
   modes: COLOR_STYLES,
   contentColors,
-  getMondayColorAsStyle: (color: string, mode = COLOR_STYLES.REGULAR, withVar = true) => {
+  getMondayColorAsStyle: (
+    color: string,
+    mode: "regular" | "hover" | "selected" = COLOR_STYLES.REGULAR,
```

Comment:
if we convert color style to be enum we could just use the enum as the type no? (instead of "regular" | "hover" | "selected")

Reply:
will change to the proper type